packages/neuron-ui/src/widgets/TextField/index.tsx

```diff
@@ -66,16 +74,27 @@ const TextField = React.forwardRef(
             onChange={onChange}
             onClick={onClick}
             readOnly={readOnly}
+            disabled={disabled}
             {...rest}
           />
           {suffix ? <span className={styles.suffix}>{suffix}</span> : null}
+          {!suffix && type === 'password' && (
+            <span
+              className={`${styles.suffix} ${styles.password}`}
+              onClick={changePasswordHide}
+              role="button"
+              aria-hidden="true"
```

Comment:
Should `aria-hidden` change with `isPasswordHide`? BTW, better to rename `isPasswordHide` to `isPasswordHidden`

Reply:
I thought for non-interactive use, there is no need to show password?

Reply:
> I thought for non-interactive use, there is no need to show password?

Then why `changePasswordHide` is bound to the element if it's non-interactive?

Reply:
I mean when disabilities peoples use it. is there no need to show the password?

Reply:
Got it.